api/prisma/seed.ts

```diff
@@ -173,4 +173,4 @@ switch (process.env.STAGE) {
             await client.prisma.$disconnect();
         });
     
-}
+}
```

Comment:
Please fix all eslint and prettier warnings.
<img width="1357" alt="image" src="https://user-images.githubusercontent.com/48569671/204015165-5e8c600d-e78d-41bc-8271-1061f4f62894.png">